.github/ISSUE_TEMPLATE/card_author_bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Card Author Bug Report
```

Comment:
Capitals?

Reply:
For the `name` field? I'm following the GitHub convention for YAML files where fields are all lowercase